bio/megahit/wrapper.py

```
@@ -0,0 +1,106 @@
+    if hasattr(snakemake.output, "contigs"):
+        output_file = snakemake.output.contigs
+    else:
+        output_file = snakemake.output[0]
+
+    output_dir = os.path.split(output_file)[0]
+
+
+# parse params
+extra = snakemake.params.get("extra", "")
+klist = snakemake.params.get("klist", "'21,29,39,59,79,99,119,141'")
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)